swagger.yaml

```diff
@@ -1235,6 +1246,27 @@ definitions:
       option:
         description: "An option for a dimension"
         type: string
+  PatchOptions:
+    description: "A list of operations to patch a dimension option. Can only handle adding values for /node_id and /order. Each element in the array is processed in sequential order."
+    type: array
+    items:
+      type: object
+      description: "Item containing all necessary information to make a single operation on the resource."
+      properties:
+        op:
+          description: |
+            The operation to be made on path.
+            * add - Sets the value for the provided path
+          type: string
+          enum: [add]
+        path:
+          description: "Path to value that needs to be operated on. The special"
```

Comment:
A typo here? Did you mean to put something on the end of `The special
`?

Reply:
good point! I don't know what else I was supposed to write here. I will remove `The special`